webex/content.js

```diff
@@ -129,43 +128,37 @@ function setText(id, text) {
     fadeBackground(e);
 }
 
+function getActiveElement(element = document.activeElement) {
+    if (element.nodeName === 'IFRAME' || element.nodeName === 'FRAME')
+        return null; // skip to allow child documents to handle shortcut
```

Comment:
Instead of using `all_frames` and handling it like this here, would it be more elegant to recurse into it in this function?

Reply:
Quite possibly, I don't recall if I tried that approach.